x-pack/plugins/apm/common/projections.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export enum PROJECTION {
```

Comment:
Not sure why anything at all in our whole codebase needs to be ALL_CAPS. I prefer the formatting of the `ProcessorEvent` enum. Just my opinion and I don't care enough to ever block merging of anything.